node/tessel-export.js

```diff
@@ -408,11 +410,16 @@ Tessel.Port.PATH = {
   'B': '/var/run/tessel/port_b'
 };
 
-Tessel.Pin = function(pin, port, interruptSupported, analogSupported) {
+Tessel.adcCompatiblePins = [4, 7];
+Tessel.pullCompatiblePins = [2, 3, 4, 5, 6, 7];
+Tessel.interruptCompatiblePins = [2, 5, 6, 7];
```

Comment:
Ah I only did this because of this comment: https://github.com/tessel/t2-firmware/pull/147#discussion_r51959053

I thought you were saying we should define them once they can easily be used in tests and the application (which I think is a very reasonable suggestion).

I'll go ahead and change them to match the `Tessel.Pin.adcCapablePins`. pattern.